windows/devices/uwb/UwbDeviceConnector.cxx

```diff
@@ -29,7 +29,9 @@ class RegisteredCallbackToken
 
 UwbConnector::UwbConnector(std::string deviceName) :
     m_deviceName(std::move(deviceName))
-{}
+{
+    NotificationListenerStart();

```

Comment:
Still doesn't seem like this is needed here. If it is, can you help me understand why?

Reply:
Missed this, thanks